File: go/template.go
package main

import (
	"fmt"
	"time"
)

func solvePuzzle(input string) int {
	return 0
}

func main() {
	const CorrectAnswer = 0
	const Input = ""

	startTime := time.Now()
	var result = solvePuzzle(Input)

	fmt.Println("--- Puzzle result ---\n ")
	fmt.Println(result)
	if result == CorrectAnswer {
		fmt.Println("Test success")
	} else {
		fmt.Println("Test fail")
	}
	fmt.Printf("Run time: %s", time.Since(startTime))
	fmt.Println("\n---------------------")
}

File: python/template.py
import time

def solvePuzzle(input):
  return 0

CORRECT_ANSWER = 0
INPUT = ""

startTime = time.time()
result = solvePuzzle(INPUT)
print('--- Puzzle result ---\n')
print(result)
print('Test success' if result == CORRECT_ANSWER else 'Test fail')
print('Run time: %s seconds' %(time.time() - startTime))
print('\n---------------------')
